eppoclient/bandits_test.go

```diff
@@ -51,16 +51,16 @@ func Test_InferContextAttributes(t *testing.T) {
 }
 
 func Test_bandits_sdkTestData(t *testing.T) {
-	flags := readJsonFile[ufcResponse]("test-data/ufc/bandit-flags-v1.json")
+	ufc := readJsonFile[ufcResponse]("test-data/ufc/bandit-flags-v1.json")
 	bandits := readJsonFile[banditResponse]("test-data/ufc/bandit-models-v1.json")
-	configStore := configurationStore{
-		flags:   flags.Flags,
-		bandits: bandits.Bandits,
-	}
+	configStore := newConfigurationStore(configuration{
+		ufc:     ufc,
```

Comment:
I feel like this should still be `flags` and `bandits`? Or bandit flags and bandit parameters. To me `ufc = flags UNION bandits`.

Reply:
Sounds good. I was confused because UFC is universal Flag configuration.

With this input, I think it doesn't make sense to use "ufc" in SDK at all as it's all UFC. I have pushed another commit renaming ufc to flags/config throughout the code.